libmamba/tests/src/specs/test_match_spec.cpp

```diff
@@ -446,9 +446,28 @@ TEST_SUITE("specs::match_spec")
         {
             auto ms = MatchSpec::parse(R"(numpy >1.8,<2|==1.7,!=1.9,~=1.7.1 py34_0)").value();
```

Comment:
I don't understand why we didn't have any error here since the support for ~= operator was missing? Was it silently ignored? 

NB: this is definitely not a blocker for getting this merged

Reply:
It was handled in context, but not for simple cases.

I think that we also need to add a test on `micromamba`.

Reply:
Done in 7056766.